readme.md

```diff
@@ -12,3 +12,33 @@ There are a couple of ways to setup the local development environment:
 
 - [Developing inside Docker](https://github.com/AutoFixture/AutoFixture.github.io/wiki/Docker-Environment-Setup)
 - [Developing locally](https://github.com/AutoFixture/AutoFixture.github.io/wiki/Local-Environment-Setup)
+
+## Roadmap 🚧
+
+> :warning: There is currently an ongoing effort to document AutoFixture.
+> Any contributions to the documentation are welcome.
+
+- [ ] Overview
+- [ ] Quick Start guide
+- [ ] Fixture creation
+- [ ] Fixture customization
+  - [ ] Customize
+  - [ ] Build
+- [ ] Freeze and Inject features
+- [ ] Testing frameworks integration
+  - [ ] xUnit
+  - [ ] NUnit
+- [ ] Mocking framework integration
+  - [ ] Moq
+  - [ ] NSubstitute
+  - [ ] FakeItEasy
+  - [ ] Foq
+  - [ ] RhinoMock
```

Comment:
I would not describe this, as RhinoMock is not being developed anymore. Actually, I would just drop support of it in v5 if you ask me :)

Reply:
agreed )